packages/reason-react/reason-react.0.12.0/opam

```diff
@@ -0,0 +1,57 @@
+opam-version: "2.0"
+synopsis: "Reason bindings for React.js"
+description: """
+ReasonReact helps you use Reason to build React components with deeply integrated, strong, static type safety.
+
+It is designed and built by people using Reason and React in large, mission critical production React codebases."""
+maintainer: [
+  "David Sancho <dsnxmoreno@gmail.com>"
+  "Antonio Monteiro <anmonteiro@gmail.com>"
+]
+authors: [
+  "Cheng Lou <chenglou92@gmail.com>" "Ricky Vetter <rickywvetter@gmail.com>"
+]
+license: "MIT"
+homepage: "https://reasonml.github.io/reason-react"
+doc: "https://reasonml.github.io/reason-react"
+bug-reports: "https://github.com/reasonml/reason-react/issues"
+depends: [
+  "dune" {>= "3.9"}
+  "ocaml" {>= "4.13.0"}
+  "melange" {>= "1.0.0"}
+  "reason-react-ppx" {= version}
+  "reason" {>= "3.6.0" & < "3.10.0"}
+  "ocaml-lsp-server" {with-test}
+  "merlin" {= "4.10-414" & with-test}
```

Comment:
shouldn't the merlin test dependency only appear on the reason-react-ppx package?

Reply:
Totally

Reply:
Thanks!